pkg/mapStructure/encoder.go

```diff
@@ -0,0 +1,29 @@
+package mapStructure
```

Comment:
can you gave a more indicative name of what the package does?
`mapStructure` is name of the external package you're using, it's not what this package does

Reply:
done